server/routes/baseClientRouter.ts

```diff
@@ -0,0 +1,26 @@
+import { RequestHandler, Router } from 'express'
+import { Services } from '../services'
+import asyncMiddleware from '../middleware/asyncMiddleware'
+import BaseClientController from '../controllers/baseClientController'
+
+export default function baseClientRouter(services: Services): Router {
+  const router = Router()
+
+  const get = (path: string | string[], ...handlers: RequestHandler[]) =>
```

Comment:
Any way of lifting these get post etc definitions out of individual router files so they can be reused?

Reply:
Hmm maybe not unless you pass router in, ignore me

Reply:
yes - I can take a pattern from approved-premises-ui but I'd like to get this in and create a separate ticket